base/mmap.jl

```diff
@@ -123,10 +123,10 @@ function mmap{T,N}(io::IO,
         name, readonly, create = settings(io)
         szfile = convert(Csize_t, len + offset)
         readonly && szfile > filesize(io) && throw(ArgumentError("unable to increase file size to $szfile due to read-only permissions"))
-        handle = create ? ccall(:CreateFileMappingW, stdcall, Ptr{Void}, (Cptrdiff_t, Ptr{Void}, Cint, Cint, Cint, Cwstring),
+        handle = create ? ccall(:CreateFileMappingW, stdcall, Ptr{Void}, (Cptrdiff_t, Ptr{Void}, Cuint, Cuint, Cuint, Cwstring),
                                 file_desc, C_NULL, readonly ? PAGE_READONLY : PAGE_READWRITE, szfile >> 32, szfile & typemax(UInt32), name) :
-                          ccall(:OpenFileMappingW, stdcall, Ptr{Void}, (Cint, Cint, Cwstring),
-                            readonly ? FILE_MAP_READ : FILE_MAP_WRITE, true, name)
+                          ccall(:OpenFileMappingW, stdcall, Ptr{Void}, (Cuint, Cuint, Cwstring),
```

Comment:
I think the second input here should stay `Cint`, ref https://msdn.microsoft.com/en-us/library/windows/desktop/aa366791(v=vs.85).aspx and https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx